GameClasses/InventoryGui.cs

```diff
@@ -107,35 +108,50 @@ internal static void PostfixUpdateRecipeList(InventoryGui __instance, List<Recip
                 //Debug.Log("Recipe list:\n");
 
                 List<KeyValuePair<Recipe, ItemDrop.ItemData>> list = new List<KeyValuePair<Recipe, ItemDrop.ItemData>>();
+
+                List<Recipe> newRecipesList = new List<Recipe>();
+
                 for (int l = 0; l < recipes.Count; l++)
                 {
-                    //Debug.Log(recipes[l].name);
-                    //Debug.Log(recipes[l].m_item.m_itemData.m_shared.m_maxQuality);
-
                     Recipe recipe2 = recipes[l];
-
                     if (recipe2.m_item.m_itemData.m_shared.m_maxQuality >= 1)
                     {
                         __instance.m_tempItemList.Clear();
-                        localPlayer.GetInventory().GetAllItems(recipe2.m_item.m_itemData.m_shared.m_name, __instance.m_tempItemList);
 
+                        if (recipe2.m_item.m_itemData.m_shared.m_maxStackSize == 1)
+                        {
+                            localPlayer.GetInventory().GetAllItems(recipe2.m_item.m_itemData.m_shared.m_name, __instance.m_tempItemList);

```

Comment:
extract localPlayer.GetInventory() to a variable and reuse instead of calling it every time